app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from '../redux/message/greetingSlice';
+
+const Greetings = () => {
+  const {
+    greeting, isFetched, isLoading, hasError,
+  } = useSelector((store) => store.greeting);
+
+  const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(fetchGreeting());
+    }, [dispatch]);
+
+
+  if (isLoading) return <h1 className="loading">... Is Loading</h1>;
+  if (hasError) return <h1 className="error">An error has ocured</h1>;
+
+  return <h1 className="greeting">{greeting}</h1>;
+};
+
```

Comment:
Just as the previous reviewer suggested, I am still getting some errors when trying to retrieve the data from the API. it displays the `is loading` state and renders a blank page. Kindly do well to fix the error.

<img width="1074" alt="image" src="https://github.com/saluumaa/hello-rails-react/assets/64643995/a131628f-6407-46fb-8376-e4b142b2edba">

But if you are sure it works on your end, you can attach a video walking us though the project👍🏼.

Reply:
I will surely provide the video because it works fine and i am not getting an error in the console as well, thanks any way @Ibizugbe